addons/mail/static/src/models/thread/thread.js

```diff
@@ -129,6 +112,27 @@ function factory(dependencies) {
         // Public
         //----------------------------------------------------------------------
 
+        /**
+         * Fetches info from a list of channels.
+         *
+         * This method is a low level wrapper for the `channel_info` RPC.
+         *
+         * @static
+         * @param {Object} param0
+         * @param {integer[]} param0.ids
```

Comment:
missing comment (I assume it's list of channel ids, but it's not super obvious at a glance)